src/attack.c

```diff
@@ -996,6 +996,17 @@ attack_network (struct arglist *globals, kb_t *network_kb)
   sched = plugins_scheduler_init
            (prefs_get ("plugin_set"), prefs_get_bool ("auto_enable_dependencies"),
             network_phase);
+  if (sched == NULL)
+    {
+      error_message_to_client (global_socket,
+                               "An NVT oid has not category assigned. This is "
+                               "considered a fatal error, since the NVTI Cache "
+                               "structure stored in Redis is out dated or "
+                               "corrupted.", NULL,
+                               NULL);
```

Comment:
This error message is too specific to one error case (missing nvt category.)

Something like "Failed to initialize the plugins scheduler." is better fit IMO.

Reply:
I agree. Changed.